modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataStorageMetricsImpl.java

```diff
@@ -187,25 +187,28 @@ public DataStorageMetricsImpl(
         MetricRegistry mreg = mmgr.registry(DATASTORAGE_METRIC_PREFIX);
 
         walLoggingRate = mreg.hitRateMetric("WalLoggingRate",
-            "Average number of WAL records per second written during the last time interval.",
+            "Average number of WAL records written per second. " +
+                "The rate is calculated over the configured time interval.",
             rateTimeInterval,
             subInts);
 
         walWritingRate = mreg.hitRateMetric(
             "WalWritingRate",
-            "Average number of bytes per second written during the last time interval.",
+            "Average number of bytes written to WAL per second. " +
+                "The rate is calculated over the configured time interval.",
             rateTimeInterval,
             subInts);
 
         walFsyncTimeDuration = mreg.hitRateMetric(
-            "WalFsyncTimeDuration",
-            "Total duration of fsync",
+            "WalFsyncDuration",
+            "Total time operations spent in the fsync phase writing WAL records to the disk " +
+                "since the start of the node in milliseconds.",
             rateTimeInterval,
             subInts);
 
         walFsyncTimeNum = mreg.hitRateMetric(
-            "WalFsyncTimeNum",
-            "Total count of fsync",
+            "WalFsyncNum",
```

Comment:
Do we really need to change metric name?
Actually metric name is kind of public API and we don't want to change it without strong reason.

Reply:
Well, the old name doesn't make much sense, so I'd rather rename it.
Personally I would rename the metric and specify the name of the new one in the release notes. WDYT?